README.md

```diff
@@ -1,3 +1,10 @@
+**Important Update**
+> The developer documentation for Adobe Commerce and Magento Open Source is moving to the [Adobe Developer site](developer.adobe.com/commerce) and [Adobe Experience League](https://experienceleague.adobe.com/docs/commerce.html). After a topic is moved, use the link in the topic header to find the new location.
```

Comment:
Adobe Experience League is also a site.
Maybe change to _Adobe Developer and Adobe Experience League sites_ for consistency?